aptos-move/framework/aptos-framework/sources/HashMap.move

```diff
@@ -0,0 +1,291 @@
+module AptosFramework::HashMap {
```

Comment:
Please describe what this does in a documentation comment. What's the difference to Table? When should a user prefer this for a Table?